clippy_lints/src/double_comparison.rs

```diff
@@ -63,6 +63,7 @@ impl<'a, 'tcx> DoubleComparisons {
                 );
             }};
         }
+        #[rustfmt::skip]
```

Comment:
This is a rustfmt bug:
it formats the match arm to drop the `{}` and with that the line exceeds the maximum length.